src/utils/consts/consts_vars.go

```diff
@@ -0,0 +1,44 @@
+package consts
+
+import (
+	"fmt"
+
+	style "github.com/NeowayLabs/semantic-release/src/utils/stdout_style"
+)
+
+const (
+	Major                         = "MAJOR"
+	Minor                         = "MINOR"
+	Patch                         = "PATCH"
+	VersionPattern                = "%d.%d.%d"
+	VersionPyVariable             = "__version__"
+	ChangeLogDefaultFile          = "CHANGELOG.md"
+	SetupPythonDefaultFile        = "setup.py"
+	CommitMsgLimitCharacters      = 150
+	AuthorChangelogFormat         = "@%s"
+	ChangeLogCommitHashLinkFormat = "[%s](https://%s/%s/%s/-/commit/%s)"
+)
+
+var (
+	CommitChangeTypes             = []string{"build", "ci", "docs", "fix", "feat", "perf", "refactor", "style", "test", "breaking change", "skip", "skip versioning", "skip v"}
+	CommitChangeTypesMajorUpgrade = []string{"breaking change"}
```

Comment:
@esequielvirtuoso `breaking change`, `breaking changes` or both? I think it could include both (if it is considered exactly)

Reply:
Yeah, we are considering exactly once it is a critical decision to make by similarity.
I'll add this option.